windows-driver-docs-pr/storage/stornvme-feature-support.md

```diff
@@ -84,3 +84,7 @@ The following matrix lists NVME features and indicates the support provided by *
 | NVMe Boot                                                      | X | |
 | Controller Fatal Status Condition                              | X | Log the event and continue with controller re-initialization |
 --->
+
+## Reference Links
+-   [Working with NVMe Devices](https://docs.microsoft.com/windows/win32/fileio/working-with-nvme-devices)
+-   [NVMe Storage Protocal Data Type](https://docs.microsoft.com/windows/win32/api/winioctl/ne-winioctl-storage_protocol_nvme_data_type)
```

Comment:
Looks OK, just a cosmetic detail to mention here:
```suggestion
- [Working with NVMe Devices](https://docs.microsoft.com/windows/win32/fileio/working-with-nvme-devices)
- [NVMe Storage Protocal Data Type](https://docs.microsoft.com/windows/win32/api/winioctl/ne-winioctl-storage_protocol_nvme_data_type)
```
(Bullet point lists only need 1 space between the source code dashes and the text after them. Any extra spacing is redundant and should be avoided, if possible.)